README.md

```diff
@@ -0,0 +1,3 @@
+# MongoDB Server blueprint
+
+This project contains entities for and other items for use with Apache Brooklyn.
```

Comment:
Add a brief description here about what this blueprint does and provide links to MongoDB resources.